include/operations/blas3_trees_gemm.hpp

```diff
@@ -1,4 +1,4 @@
-/*************************************************************************** 
+/***************************************************************************
  *  @license
  *  Copyright (C) 2017 Codeplay Software Limited
  *  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This file was not intended to be changed. I will consider resubmitting the pull request.